<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color:rgb(5, 143, 223)</string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1051</width>
      <height>791</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1033</width>
       <height>773</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(46, 127, 214)</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>501</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="player1Layout">
       <property name="spacing">
        <number>-1</number>
       </property>
      </layout>
     </widget>
     <widget class="QLabel" name="player1Name">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);font: 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>Player1:</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="player2Name">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);font: 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>Player2:</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>90</y>
        <width>481</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="player2Layout"/>
     </widget>
     <widget class="QLabel" name="player3Name">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>290</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);font: 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>Player3:</string>
      </property>
     </widget>
     <widget class="QLabel" name="player4Name">
      <property name="geometry">
       <rect>
        <x>558</x>
        <y>290</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);font: 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>Player4:</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>501</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="player3Layout"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>340</y>
        <width>481</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="player4Layout"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>560</y>
        <width>501</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="player5Layout"/>
     </widget>
     <widget class="QLabel" name="player5Name">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>510</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);font: 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>Player5:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>260</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="player2Hit">
         <property name="text">
          <string>Hit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="player2Fold">
         <property name="text">
          <string>Fold</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>500</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="player3Hit">
         <property name="text">
          <string>Hit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="player3Fold">
         <property name="text">
          <string>Fold</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>500</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="player4Hit">
         <property name="text">
          <string>Hit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="player4Fold">
         <property name="text">
          <string>Fold</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>730</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="player5Hit">
         <property name="text">
          <string>Hit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="player5Fold">
         <property name="text">
          <string>Fold</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="player1timer">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);font: 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>10 sec</string>
      </property>
     </widget>
     <widget class="QLabel" name="player2timer">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);font: 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>10 sec</string>
      </property>
     </widget>
     <widget class="QLabel" name="player3timer">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);font: 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>10 sec</string>
      </property>
     </widget>
     <widget class="QLabel" name="player4timer">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>290</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);font: 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>10 sec</string>
      </property>
     </widget>
     <widget class="QLabel" name="player5timer">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>510</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255);font: 24pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string>10 sec</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>381</x>
        <y>260</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="player1Hit">
         <property name="text">
          <string>Hit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="player1Fold">
         <property name="text">
          <string>Fold</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>horizontalLayoutWidget</zorder>
     <zorder>player1Name</zorder>
     <zorder>player2Name</zorder>
     <zorder>horizontalLayoutWidget_2</zorder>
     <zorder>player3Name</zorder>
     <zorder>player4Name</zorder>
     <zorder>horizontalLayoutWidget_3</zorder>
     <zorder>horizontalLayoutWidget_4</zorder>
     <zorder>horizontalLayoutWidget_5</zorder>
     <zorder>player5Name</zorder>
     <zorder>pushButton</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>player1Fold</zorder>
     <zorder>player1Hit</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget_2</zorder>
     <zorder>layoutWidget_3</zorder>
     <zorder>layoutWidget_4</zorder>
     <zorder>player1timer</zorder>
     <zorder>player2timer</zorder>
     <zorder>player3timer</zorder>
     <zorder>player4timer</zorder>
     <zorder>player5timer</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
